Athos/TFEzPCLibrary/Library64_cpp.ezpc

```diff
@@ -297,6 +529,29 @@ def void ReduceMean24(int32_pl outS1, int32_pl outS2,
 	};
 }
 
+(* This one is used for onnx compilation *)

```

Comment:
Minor: For consistency, can we add (*****...*) before this new function we are adding. Also, can we remove the comment about onnx compilation? This is anyway generic no?